cmd/keymasterd/testdata/make-test-data

```diff
@@ -0,0 +1,65 @@
+#! /bin/bash --posix
+#
+# make-test-data
+#
+# A simple script to generate test data (CAs and certs). This is normally run
+# just once, as the testfiles are expected to be checked into the repository.
+# This script is idempotent: it will not overwrite files.
+
+set -eu
+set -o pipefail
+
+make_key ()
+{
+    local -r name="$1"
+
+    if [ ! -r "$name.key" ]; then
+	openssl genpkey -algorithm RSA -out "$name.key" \
+		-pkeyopt rsa_keygen_bits:512
```

Comment:
2048 please

Reply:
As per Slack discussion, bumping to 1024. We don't want to make these too large otherwise TravisCI will slow down.